katib/v1alpha1/base/katib-ui-deployment.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: katib-ui
+  namespace: kubeflow
+  labels:
+    app: vizier
+    component: ui
+spec:
+  replicas: 1
+  template:
+    metadata:
+      name: katib-ui
+      labels:
+        app: vizier
+        component: ui
+    spec:
+      containers:
+      - name: katib-ui
+        image: gcr.io/kubeflow-images-public/katib/katib-ui:v0.1.2-alpha-156-g4ab3dbd
```

Comment:
should this be a parameter?

Reply:
We can do `kustomize edit set namespace <new namespace>` inside the `katib/v1alpha1/base` folder to set it to a different namespace.

Reply:
it can be overwritten by same image name in images list of kustomization.yaml